docs/src/tutorials/speech-brain.rst

```diff
@@ -0,0 +1,94 @@
+********************
+SpeechBrain
+********************
+
+In this short tutorial, we're going to demonstrate how Oríon can be integrated to a `SpeechBrain
+<https://speechbrain.github.io/>`_ speech recognition model.
+The files mentioned in this tutorial are available in the `Oríon
+<https://github.com/Epistimio/orion/tree/develop/examples>`_ repository.
+
+Installation and setup
+======================
+
+Make sure Oríon is installed (:doc:`/install/core`).
+
+Then install SpeechBrain using ``$ pip install speechbrain``
+
+Adapting the Speechbrain for Oríon
+==================================
+
+The Adaptation for using Oríon is quite simple.
```

Comment:
It's not clear from which SpeechBrain tutorial you are starting off. You should point to it here so that we better understand the changes that are done to the original version.

Reply:
Done in 272a8186047d7ffde1732a8e187f885aaf75e5eb, Thanks!